pylint/lint/parallel.py

```diff
@@ -79,7 +83,7 @@ def _worker_check_single_file(file_item):
     return (
         id(multiprocessing.current_process()),
         _worker_linter.current_name,
-        filepath,
+        file_item[1],
```

Comment:
```suggestion
        file_item.filepath,
```
One advantage of using a NamedTuple.